shared/src/main/scala/drt/shared/api/Arrival.scala

```diff
@@ -91,7 +91,7 @@ case class Arrival(Operator: Option[Operator],
     (minutesToDisembark * oneMinuteInMillis).toLong
   }
 
-  val bestPaxEstimate: Int = PcpPax.bestPaxEstimate(this)
+  val bestPaxEstimate: Int = PcpPax.bestPcpPaxEstimate(this)
```

Comment:
Could be worth renaming this val to `bestPcpPaxEstimate` to make it clearer what it's returning